storybook/stories/UserInput/Chip.stories.tsx

```diff
@@ -4,39 +4,75 @@ import { Chip } from "@src";
 import CenterView from "../../helpers/CenterView.react";
 import { withKnobs } from "@storybook/addon-knobs";
 import { View } from "react-native";
-
+import { Colors } from "react-native-ameelio-library";
```

Comment:
ahhh so unfortunately this won't work because it'll import circularly once we try to build + publish to npm. instead, you should `import * as Colors from '../Brand/Colors'` or I think you can also do `import {Colors} from '../Styles';`